harness/determined/core/_train.py

```diff
@@ -16,6 +19,27 @@ class EarlyExitReason(enum.Enum):
     USER_REQUESTED_STOP = "EXITED_REASON_USER_REQUESTED_STOP"
 
 
+class TensorboardSyncMode(enum.Enum):
+    """
+    ```TensorboardSyncMode`` defines how Tensorboard metrics and profiling data are retained.
+
+    In ``ChiefOnly`` mode only the chief uploads its metrics and profiling data to checkpoint
+    storage. This is the same behavior that existed prior to 0.18.2.
+
+    In ``ChiefAndWorkers`` mode the chief uploads it metrics and profiling data to checkpoint
+    storage, and the workers upload their profiling data. The profiling data file names are
+    altered to include the workers' rank in order to avoid collision and allow displaying each
+    worker's profile under a separate host in Tensorboard.
+
+    In ``Manual`` mode neither metrics nor profiling data are reported. It is up to the user
+    to call ``TrainContext.upload_tb_files()``
+    """
+
+    ChiefOnly = "CHIEF_ONLY"
+    ChiefAndWorkers = "CHIEF_AND_WORKERS"
```

Comment:
Why would a user want ChiefOnly, rather than just ChiefAndWorkers?

I mean, why not just have Auto and Manual, where Auto is really ChiefAndWorkers?

Reply:
I was thinking of giving an option to have the exact same behavior as before 0.18.1 but I guess there is little value in that?